### DISCUSSION TITLE
有没有方法将DataGrid的列横向显示，行纵向显示？

### DISCUSSION THREAD

**Discussion by wenxuan25** [BYJRK/DotNet-Discussions#131] [WPF 开发]
最近遇到一个问题，由于DataGrid的列Binding太多数据，导致列太多，行太少，有没有什么好方法行列交换显示

**Comment by BYJRK** [comment]
搜了一篇文章
https://www.grapecity.com/blogs/4-steps-to-transpose-and-invert-wpf-datagrid

**Reply by ccq1** [reply]
牛牛牛

**Reply by wenxuan25** [reply]
好像收费

**Comment by ivei** [comment]
https://github.com/GuOrg/Gu.Wpf.DataGrid2D
看下这.

**Reply by ivei** [reply]
![image](https://github.com/BYJRK/DotNet-Discussions/assets/22268333/eb9f5d2d-f88c-4b6f-8d40-7f2ba03f06f5)
